app/views/admin/creator_settings/new.html.erb

```diff
@@ -7,37 +7,23 @@
     <div aria-live="assertive">
       <% if flash[:error] %>
         <div class="crayons-notice crayons-notice--danger mb-6" role="alert">
-          <%= flash[:error] %>
+          <% flash[:error].each do |message| %>
+            <div><%= message %></div>
+          <% end %>
         </div>
       <% end %>
     </div>
 
-    <%= form_tag(admin_creator_settings_path, method: "post", multipart: true, class: "relative z-elevate p-4", "data-action": "submit->creator-settings#formValidations") do %>
-      <% if defined?(resource) && resource&.errors&.any? %>
-        <div class="crayons-card crayons-card--secondary crayons-notice crayons-notice--danger" role="alert" data-testid="signup-errors">
-          <div class="crayons-card__header">
-            <h1 class="crayons-card__headline">
-              Whoops, we found <%= pluralize(resource.errors.size, "problem") %>
-            </h1>
-          </div>
-          <div class="crayons-card__body">
-            <ul>
-              <% resource.errors.full_messages.each do |message| %>
-                <li><%= message %></li>
-              <% end %>
-            </ul>
-          </div>
-        </div>
-      <% end %>
+    <%= form_with url: admin_creator_settings_path, model: @creator_settings_form, local: true, method: "post", multipart: true, class: "relative z-elevate p-4", "data-action": "submit->creator-settings#formValidations" do |f| %>
```

Comment:
We user the `creator_settings_form` model to render the form. In addition, now when there is an error, the values that were saved already pull through in the form.